UPGRADE-4.x.md

```diff
@@ -1,9 +1,15 @@
 UPGRADE 4.x
 ===========
 
-UPGRADE FROM 3.x to 3.x
+UPGRADE FROM 4.x to 4.x
 =======================
 
+### Upgrade to SonataDatagridBundle 3.0
+
```

Comment:
With this approach, we should later group also the required changes for "nelmio/api-doc-bundle" in this section.
IMO, we should stop using bundle aliases (like "SonataDatagridBundle") in order to be specific about the package names.
```suggestion
### Dependencies

 - "sonata-project/datagrid-bundle" is bumped from ^2.4 to ^3.0.
```